### PR TITLE
blog: Add A poor man API post

### PR DESCRIPTION
**Pull request by nfrankel**

### CODE REVIEW

**Comment by guitu168** [comment]
image

Preview page is bad. cc @SkyeYoung

File: blog/en/blog/2022/11/23/poor-man-api.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+    image_url: https://avatars.githubusercontent.com/u/752258
+keywords:
+  - API
+  - PostgreSQL
+  - PostgREST
+  - no code
+  - MVP
+description: Creating a full-fledged API requires resources, both time and money. You need to think about the model, the design, the REST principles, etc., without writing a single line of code. Most of the time, you don't know whether it's worth it: you'd like to offer a Minimum Viable Product and iterate from there. I want to show how you can achieve it without writing a single line of code.
+tags: [Case Studies]
+image: https://blog.frankel.ch/assets/resources/poor-man-api/jar-g19c1e6973.jpg
```
Comment (guitu168) [inline blog/en/blog/2022/11/23/poor-man-api.md:16]: This link is invalid. I can't access it.

File: blog/en/blog/2022/11/23/poor-man-api.md [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+---
+
+> Creating a full-fledged API requires resources, both time and money. You need to think about the model, the design, the REST principles, etc., without writing a single line of code. Most of the time, you don't know whether it's worth it: you'd like to offer a Minimum Viable Product and iterate from there. I want to show how you can achieve it without writing a single line of code.
+
+<!--truncate-->
+
+<head>
+    <link rel="canonical" href="https://blog.frankel.ch/poor-man-api/" />
```
Comment (guitu168) [inline blog/en/blog/2022/11/23/poor-man-api.md:24]: ditto

File: blog/en/blog/2022/11/23/poor-man-api.md [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+    title: Author
+    url: https://github.com/nfrankel
+    image_url: https://avatars.githubusercontent.com/u/752258
+keywords:
+  - API
+  - PostgreSQL
+  - PostgREST
+  - no code
+  - MVP
+description: Creating a full-fledged API requires resources, both time and money. You need to think about the model, the design, the REST principles, etc., without writing a single line of code. Most of the time, you don't know whether it's worth it: you'd like to offer a Minimum Viable Product and iterate from there. I want to show how you can achieve it without writing a single line of code.
```
Comment (SkyeYoung) [inline blog/en/blog/2022/11/23/poor-man-api.md]:
```suggestion
description: "Creating a full-fledged API requires resources, both time and money. You need to think about the model, the design, the REST principles, etc., without writing a single line of code. Most of the time, you don't know whether it's worth it: you'd like to offer a Minimum Viable Product and iterate from there. I want to show how you can achieve it without writing a single line of code."
```

**Comment by nfrankel** [comment]
@hf400159 Links are not available because I didn't publish the blog post *yet*. You should have everything on Monday morning when you wake up

**Review by SylviaBABY — APPROVED** [review]

**Review by SkyeYoung — APPROVED** [review]

**Pull request merged**